src/app/study/page.tsx

```
@@ -0,0 +1,86 @@
+import { CategoryCard } from "@/components/citizenship/category-card";
+import { Card, CardContent, CardDescription, CardHeader, CardTitle } from "@/components/ui/card";
+import { getAllCategories, getQuestionsByCategory, getRandomQuestions } from "@/lib/sample-questions";
+import { BookOpen, ChevronRight } from "lucide-react";
+import Link from "next/link";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)